docs/src/main/asciidoc/spanner.adoc

```diff
@@ -404,6 +404,22 @@ A table may have multiple child tables.
 On updating or inserting an object to Cloud Spanner, all of its referenced children objects are also updated or inserted in the same request, respectively.
 On read, all of the interleaved child rows are also all read.
 
+===== Lazy Fetch
+
+`@Interleaved` properties are retrieved eagerly by default, but can be fetched lazily for performance in both read and write:
+
+[source,java]
+----
+@Interleaved(lazy = true)
```

Comment:
Are there some existing enums we can reuse, like `fetch = FetchType.LAZY`?

Reply:
Yes, however they are not in any dependency we have. 

I think it is for this reason that spring data Mongodb uses a boolean like we do here.